CODEOWNERS

```
@@ -0,0 +1 @@
+* @justincormack @SteveLasker
```

```suggestion
* @notaryproject
```

This will allow anyone part of this team to own the code.